cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-media.php

```diff
@@ -1741,9 +1745,12 @@ public function maybe_overwrite_featured_image( $attachment_id ) {
 	 */
 	public function set_doing_featured( $post_id, $attachment_id ) {
 		$this->doing_featured_image = $attachment_id;
-		add_action( 'end_fetch_post_thumbnail_html', function () {
-			$this->doing_featured_image = false;
-		} );
+		add_action(
```

Comment:
@dugajean can we try not to put everything on multi-line? This makes things difficult to read and creates unnecessary code changes.
Multi-line is more for JS and not a requirement in PHP.

Reply:
@dugajean this is not resolved. It's not a would be nice. it's a requirement.

Reply:
PHPCS is giving me an error for this. When we setup lint checking on pre-commit and CI, this would not pass.

Why is this a requirement?

Reply:
What ruleset are you using for your pre-commit hook?